docs/rules/multiline-html-element-content-newline.md

```diff
@@ -65,11 +65,14 @@ This rule enforces a line break before and after the contents of a multiline ele
 ```json
 {
     "vue/multiline-html-element-content-newline": ["error", {
+        "ignoreWhenEmpty": true,
         "ignores": ["pre", "textarea"]
     }]
 }
 ```
 
+- `ignoreWhenEmpty` ... allows having contents in one line, when given element has no content.
```

Comment:
this message is wierd

Reply:
Perhaps `disables reporting when element has no content`?